Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -169,6 +170,64 @@ function Run_IMB_MPI1() {
 	fi
 }
 
+function Run_OSU_P2P() {
+	total_attempts=$(seq 1 1 $imb_mpi1_tests_iterations)
+	if [[ $osu_p2p_tests == "all" ]]; then
+		extra_params=""
+	else
+		extra_params=$osu_p2p_tests
+	fi
+	for attempt in $total_attempts; do
+		LogMsg "OSU P2P test iteration $attempt for $mpi_type- Running."
+		case "$mpi_type" in
+			ibm)
+				LogMsg "Test not yet available"
+				# LogMsg "$mpi_run_path -hostlist $modified_slaves:$VM_Size -np $(($VM_Size * $total_virtual_machines)) -e MPI_IB_PKEY=$MPI_IB_PKEY $osu_p2p_path allreduce"
+				# $mpi_run_path -hostlist $modified_slaves:$VM_Size -np $(($VM_Size * $total_virtual_machines)) -e MPI_IB_PKEY=$MPI_IB_PKEY $osu_p2p_path allreduce > OSU-P2P-AllNodes-output-Attempt-${attempt}.txt
+			;;
+			open)
+				LogMsg "Test not yet available"
+				# LogMsg "$mpi_run_path --allow-run-as-root --host $master,$slaves -n $(($mpi1_ppn * $total_virtual_machines)) $mpi_settings $osu_p2p_path $extra_params"
+				# $mpi_run_path --allow-run-as-root --host $master,$slaves -n $(($mpi1_ppn * $total_virtual_machines)) $mpi_settings $osu_p2p_path $extra_params > OSU-P2P-AllNodes-output-Attempt-${attempt}.txt
+			;;
+			hpcx)
+				LogMsg "Test not yet available"
+				# LogMsg "$mpi_run_path --allow-run-as-root -x UCX_IB_PKEY=$UCX_IB_PKEY -n $(($mpi1_ppn * $total_virtual_machines)) --H $master,$slaves $mpi_settings $osu_p2p_path $extra_params"
+				# $mpi_run_path --allow-run-as-root -x UCX_IB_PKEY=$UCX_IB_PKEY -n $(($mpi1_ppn * $total_virtual_machines)) --H $master,$slaves $mpi_settings $osu_p2p_path $extra_params > OSU-P2P-AllNodes-output-Attempt-${attempt}.txt
+			;;
+			intel)
+				LogMsg "Test not yet available"
+				# LogMsg "$mpi_run_path -hosts $master,$slaves -ppn $(($VM_Size / $total_virtual_machines)) -n $(($VM_Size * $total_virtual_machines)) $mpi_settings $osu_p2p_path $extra_params"
+				# $mpi_run_path -hosts $master,$slaves -ppn $(($VM_Size / $total_virtual_machines)) -n $(($VM_Size * $total_virtual_machines)) $mpi_settings $osu_p2p_path $extra_params > OSU-P2P-AllNodes-output-Attempt-${attempt}.txt
+			;;
+			mvapich)
+				LogMsg "$osu_p2p_tests for $mpi_type"
+				LogMsg "$mpi_run_path -n $(($mpi1_ppn * $total_virtual_machines)) $master $slaves_array $mpi_settings $osu_p2p_path $extra_params"
+				ssh root@${master} "$mpi_run_path -n $(($mpi1_ppn * $total_virtual_machines)) $master $slaves_array $mpi_settings $osu_p2p_path$extra_params> OSU-P2P-AllNodes-output-Attempt-${attempt}.txt"
+			;;
+		esac
+		mpi_status=$?
+		if [ $mpi_status -eq 0 ]; then
+			LogMsg "OSU P2P test iteration $attempt - Succeeded."
+			sleep 1
```

Comment:
Use a single 'sleep 1' out of if-statement

Reply:
done